libpod/container_attach_linux.go

```diff
@@ -109,8 +109,8 @@ func (c *Container) attachContainerSocket(resize <-chan remotecommand.TerminalSi
 	case err := <-receiveStdoutError:
 		return err
 	case err := <-stdinDone:
-		if _, ok := err.(utils.DetachError); ok {
-			return nil
+		if err == ErrDetach {
```

Comment:
Do we need to do errors.Cause(err)?

Reply:
Should be guaranteed that it's not wrapped, but can't hurt